packages/core/src/components/steps-modal/steps-modal.spec.ts

```diff
@@ -43,8 +43,6 @@ describe('atom-steps-modal', () => {
             has-footer=""
             has-divider=""
             header-title="Step 1"
-            disable-secondary="false"
```

Comment:
we should validate if it is being passed to the atom-modal, man

Reply:
right! i created a test here [508ed2d](https://github.com/juntossomosmais/atomium/pull/609/commits/508ed2dd19c46b1a866caf7985d684b3db0b2205)